google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -23,7 +23,8 @@
  */
 public enum AppEngineEnvironment {
   APP_ENGINE_STANDARD("appengine.environment.name.standard"),
-  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+  APP_ENGINE_FLEX("appengine.environment.name.flexible"),
+  APP_ENGINE_FLEX_COMPAT("appengine.environment.name.flexible");
```

Comment:
tbh I find it a little confusing. E.g. I'm not sure what you mean by it "contemplating two environments". Or what you mean by us "considering" a third environment.

Essentially, in its current state, this is what I see this class being (wording is rough):

> Specifies the type of project deployable to the Google App Engine environments.

> APP_ENGINE_STANDARD: exploded war projects deployable to the App Engine Standard Environment.
APP_ENGINE_FLEX_COMPAT: like APP_ENGINE_STANDARD projects (also exploded wars, also use appengine-web.xml cofig, etc.), but are deployable to the App Engine Flexible environment.
APP_ENGINE_FLEXIBLE: those projects that do not have an appengine-web.xml, have an app.yaml config, are any kind of Java jar/war project, and are deployable to the App Engine Flexible environment